src/test/java/core/basesyntax/FruitTransactionTest.java

```diff
@@ -0,0 +1,33 @@
+package core.basesyntax;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import org.junit.jupiter.api.Test;
+
+class FruitTransactionTest {
+    @Test
+    void fromCode_InvalidCode_NotOk() {
+        String wrongCode = "balance";
+
+        assertThrows(RuntimeException.class,
+                () -> FruitTransaction.fromCode(wrongCode),
+                "Invalid operation code");
+    }
+
+    @Test
+    void fromCode_Wrong_NotOk() {
+        String wrongCode = "balance";
+
+        assertThrows(RuntimeException.class,
+                () -> FruitTransaction.fromCode(wrongCode),
+                "Invalid operation code");
+    }

```

Comment:
You have two identical test cases. Also, you should assert IllegalArgumentException